hydra-node/src/Hydra/Chain.hs

```diff
@@ -129,6 +129,8 @@ data PostTxError tx
     -- NOTE: PlutusDebugInfo does not have much available instances so we put it
     -- in Text form but it's lame
     PlutusValidationFailed {plutusFailure :: Text, plutusDebugInfo :: Text}
+  | -- | User tried to commit more than 100 ADA hardcoded limit on mainnet
+    ReachedMainnetHardcodedLimit
```

Comment:
Could: use a different name, `ReachedMainnetHardcodedLimit` means we reached the limit, not exceeded it. Also it is not saying what is limited.

A better name IMO: `CommittedTooMuchADAForMainnet`